src/util.h

```diff
@@ -682,6 +682,31 @@ enum ESkillLevel
     SL_PROFESSIONAL = 3
 };
 
+inline QString SkillLevelToString ( const int iSkill )
+{
+    switch ( iSkill )
+    {
+    case SL_NOT_SET:
+        return QCoreApplication::translate ( "CServerDlg", "None" );
+        break;
+
+    case SL_BEGINNER:
+        return QCoreApplication::translate ( "CServerDlg", "Beginner" );
+        break;
+
+    case SL_INTERMEDIATE:
+        return QCoreApplication::translate ( "CServerDlg", "Intermediate" );
+        break;
+
+    case SL_PROFESSIONAL:
+        return QCoreApplication::translate ( "CServerDlg", "Expert" );
+        break;
+
+    default:
+        return QString ( "" );
+    }
```

Comment:
```suggestion
    return iSkill;
```

I'd rather remove this.

Reply:
Please, this is better.